ESMF/UFS/Aerosol_Cap.F90

```diff
@@ -311,14 +311,16 @@ subroutine ModelDataInitialize(model, rc)
     end if
 
     ! startup MAPL
-    allocate(is % wrap % maplCap, stat=stat)
+    allocate(is % wrap % maplCap, stat=stat, source = MAPL_Cap("MAPL Driver", aerosol_routine_SS, cap_options=maplCapOptions, rc=rc))
```

Comment:
I don't think `stat` is the right variable to check here.   Should be the `rc` code from the previous call.     Please also check the other calls to the log error routines.

Reply:
After this call, both stat and rc are checked.  It should be fine